examples/advanced/sklearn-svm/utils/prepare_job_config.py

```diff
@@ -187,6 +194,7 @@ def _update_meta(meta: dict, args):
 def _update_client_config(config: dict, args, site_name: str, site_indices):
     # update client config
     # data path and training/validation row indices
+    config["components"][0]["args"]["backend"] = args.backend
```

Comment:
We might be able to enhance this later, instead of the hard-coding "0" index here.
But we can do that in a separate PR.

Reply:
Agree, we can recognize this component with its name, will enhance in a separate PR